packages/keystone/fields/ExternalContent/views/Field.js

```
@@ -0,0 +1,119 @@
+                value={editValue}
+                onChange={handleChange}
+                disabled={isDisabled}
+                css={{
+                    width: '100%',
+                    minHeight: '200px',
+                    fontFamily: 'monospace',
+                    fontSize: '12px',
+                    padding: '8px',
+                    border: '1px solid #ccc',
+                    borderRadius: '4px',
+                }}
```

ESLint

[error] Unknown property 'css' found

(react/no-unknown-property)